docs/sources/flow/getting-started/migrating-from-prometheus.md

```diff
@@ -0,0 +1,187 @@
+>
+> Replace `OUTPUT_CONFIG_PATH` with the full path to output the flow configuration.
+
+The following example demonstrates converting a Prometheus configuration:
+
+```yaml
+global:
+  scrape_timeout:    45s
+
+scrape_configs:
+  - job_name: "prometheus"
+    static_configs:
+      - targets: ["localhost:12345"]
+
+remote_write:
+  - name: "grafana-cloud"
+    url: "https://prometheus-us-central1.grafana.net/api/prom/push"
+    basic_auth:
+      username: USERNAME
+      password: PASSWORD
+```
+
+Execute:
+
+```bash
+grafana-agent convert --format=prometheus --output=OUTPUT_CONFIG_PATH INPUT_CONFIG_PATH
+```
+
+The contents of the new flow configuration:
+
+```river
+prometheus.scrape "prometheus" {
+	targets = [{
+		__address__ = "localhost:12345",
+	}]
+	forward_to     = [prometheus.remote_write.default.receiver]
+	job_name       = "prometheus"
+	scrape_timeout = "45s"
+}
+
+prometheus.remote_write "default" {
+	endpoint {
+		name = "grafana-cloud"
+		url  = "https://prometheus-us-central1.grafana.net/api/prom/push"
+
+		basic_auth {
+			username = "USERNAME"
+			password = "PASSWORD"
+		}
+
+		queue_config {
+			capacity             = 2500
+			max_shards           = 200
+			max_samples_per_send = 500
+		}
+
+		metadata_config {
+			max_samples_per_send = 500
+		}
+	}
+}
+```
+
+#### Debugging
+
+If Prometheus configuration is provided that cannot be converted,
+diagnostic information is printed to `stderr`. You can bypass
+any non-critical issues and output the flow configuration using best
+effort conversion by including the `--bypass-errors` flag.
+   
+> Be aware that the behavior may not match when bypassing errors.
+
+```bash
+grafana-agent convert --format=prometheus --bypass-errors --output=OUTPUT_CONFIG_PATH INPUT_CONFIG_PATH
+```
+
+You can also output a diagnostic report by including the `--report` flag.
+
+```bash
+grafana-agent convert --format=prometheus --report=OUTPUT_REPORT_PATH --output=OUTPUT_CONFIG_PATH INPUT_CONFIG_PATH
+```
+
+> Replace `OUTPUT_REPORT_PATH` with the full path to output the report to.
+
+Using the example Prometheus configuration from above outputs a diagnostic
+report:
+
+```
+(Info) Converted scrape_configs job_name "prometheus" into...
+  A prometheus.scrape.prometheus component
+(Info) Converted 1 remote_write[s] "grafana-cloud" into...
+  A prometheus.remote_write.default component
+```
+
+## Run a Prometheus configuration
+
+If you’re not ready to switch over to flow configuration, you can also run
+the Prometheus configuration without having to save it as a flow config.
+This allows you to try flow mode without having to modify your existing
+Prometheus configuration infrastructure.
+
+> In this task, we will use the [run][] CLI command to output flow
+> configuration from a Prometheus configuration.
+
+1. Execute the following to start the Grafana Agent in flow mode with a
+Prometheus configuration:
+
+```bash
+grafana-agent run --config.format=prometheus INPUT_CONFIG_PATH
+```
```

Comment:
I'm worried about this one. Rather than the guide instructing the user to manually invoke the `run` command, I think we should be telling them to modify their existing installation and change the command line arguments there. 

As it is now, this guide feels more like a one-off rather than actually setting up the agent to use a Prometheus config file.

Reply:
good point, specific run commands were removed throughout the whole doc now and refer to the other page and using the flags only.